Recruit-Training/Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -27,7 +27,19 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed();
 
-    // TODO: Print out time until the battery is fully charged or depleted.
+    // TODO: Print out time until the battery is fully charged or depleted.    
+    int ms = batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();
```

Comment:
Make sure that your variable names are more descriptive. Can you change ms to something more specific instead like milliseconds.